### PR TITLE
Add simple tutorial for Quicwkit on Lambdas

### PR DESCRIPTION
**Pull request by fmassot**

### CODE REVIEW

File: docs/get-started/tutorials/tutorial-aws-lambda-simple.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+It should respond under 1 second and return 10 hits out of 345 if you indexed the whole dataset. If you index the first 10,000 documents, you won't have any hits, try to query `INFO` logs instead.
+
+
+Let's now run a more advanced query: a date histogram with a term aggregation on the `severity_text`` field:
+
+```bash
+python cli.py search '{ "query": "*", "max_hits": 0, "aggs": { "events": { "date_histogram": { "field": "timestamp", "fixed_interval": "30d" }, "aggs": { "log_level": { "terms": { "size": 10, "field": "severity_text", "order": { "_count": "desc" } } } } } } }'
+```
+
+It should respond under 2 seconds and return the top log levels per 30 days.
```
Comment (rdettai) [inline docs/get-started/tutorials/tutorial-aws-lambda-simple.md:113]: Should we show the cache capability by running this query twice?

Comment (fmassot) [reply docs/get-started/tutorials/tutorial-aws-lambda-simple.md:113]: This is interesting indeed, let's do it after the blog post release.

File: docs/get-started/tutorials/tutorial-aws-lambda-simple.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+### Install AWS CDK
+
+We will use [AWS CDK](https://aws.amazon.com/cdk/) for our infrastructure automation script. Install it using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm):
+```bash
+npm install -g aws-cdk 
+```
+We also use the `curl` and `make` commands. For instance on Debian based distributions:
+```bash
+sudo apt update && sudo apt install curl make
+```
```
Comment (rdettai) [inline docs/get-started/tutorials/tutorial-aws-lambda-simple.md:28]: We actually don't use these.

File: docs/get-started/tutorials/tutorial-aws-lambda-simple.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+Finally, clone the Quickwit repository:
+```bash
+git clone https://github.com/quickwit-oss/tutorials.git
+cd tutorials/simple-lambda-stack
+```
+
+### Setup python environment
+
+We use python to define the AWS CloudFormation stack we need to deploy, and a python CLI to invoke Lambdas.
+Let's install those few packages (boto3, aws-cdk-lib, click, pyyaml).
```
Comment (rdettai) [inline docs/get-started/tutorials/tutorial-aws-lambda-simple.md:41]: Maybe explicit the requirement on Python3.10?

**Pull request merged**